### ISSUE TITLE
Regarding the execution speed and model size after   Sparsifying ResNet-50 

### ISSUE THREAD

**Issue by zirid** [neuralmagic/sparseml#2164]
I tried the same Methode described in : https://github.com/neuralmagic/sparseml/blob/main/integrations/torchvision/tutorials/docs-torchvision-sparsify-from-scratch-resnet50-beans.ipynb  with the same recipe files and here are my results: 

  
*dense

Val Loss: 0.03551835000926499
Top 1 Acc: 0.9824561403508771
model size: 89.895MB
Exec time: 0.007969 sec    (inference on  [1,3,224,224]) gpu

*sparse 90%

Val Loss: 0.20316438491152988
Top 1 Acc: 0.9883040935672515
model size: 89.895MB
Exec time: 0.009973 sec    (inference on  [1,3,224,224]) gpu

*sparse 70%

Val Loss: 0.00046509542728525173
Top 1 Acc: 1.0
model size: 89.895MB
Exec time: 0.006981    (inference on  [1,3,224,224]) gpu

Code used to get the model Size : 

```
param_size = 0
    for param in model.parameters():
        param_size += param.nelement() * param.element_size()
    buffer_size = 0
    for buffer in model.buffers():
        buffer_size += buffer.nelement() * buffer.element_size()

    size_all_mb = (param_size + buffer_size) / 1024 ** 2
    print('model size: {:.3f}MB'.format(size_all_mb))
```

I have few questions : 

1) Why the model has the same size after pruning?
2) The execution time on gpu is supposed to drop after  sparsifying the model ? 

Thank you.

**Comment by mgoin** [comment]
Hi @zirid PyTorch doesn't support unstructured sparsity so SparseML can't offer acceleration on it's own. For sparse performance, we ask that you export the model to ONNX and run with DeepSparse. Here is a section of the benchmarking guide there that shows you can get sparse model speedup with DeepSparse https://github.com/neuralmagic/deepsparse/blob/main/docs/user-guide/deepsparse-benchmarking.md#batch-sizes
